routing/scooter.go

```diff
@@ -11,14 +11,16 @@ import (
 
 var scooterService *services.ScooterService
 var scooterGrpcService *services.GrpcScooterService
+var orderService *services.OrderService
 var scooterIDKey = "scooterId"
 
 // TODO make dynamic access to this variables from UI
 //var choosenWay = models.Coordinate{Latitude: 48.4221, Longitude: 35.0196}
-//var choosenWay = models.Coordinate{ Latitude: 48.42543, Longitude: 35.02183}  // dafi
+var choosenWay = models.Coordinate{Latitude: 48.42543, Longitude: 35.02183} // dafi
 //var choosenWay = models.Coordinate{48.42272,35.02280} // visokovoltnaya
-var choosenWay = models.Coordinate{Latitude: 48.42367 , Longitude: 35.04436} // ostapa vishni
-var choosenScooter = 1
+//var choosenWay = models.Coordinate{Latitude: 48.42367 , Longitude: 35.04436} // ostapa vishni
+var choosenScooterID = 1
+var userFromRequest = models.User{ID: 1, LoginEmail: "guru_admin@guru.com", UserName: "Guru", UserSurname: "Sadh"}
```

Comment:
Did you try to get user from context? (the thing, that Alexey had shown us before)